package.json

```diff
@@ -10,6 +10,8 @@
     "serve": "ws --spa index.html -d ./dist",
     "stylelint": "stylelint './src/**/*.scss' ",
     "stylelint:fix": "stylelint './src/**/*.scss' --fix",
+    "test": "npm run test:unit",
+    "test:unit": "ts-node -r ignore-styles tests/unit/index.ts",
```

Comment:
I noticed that we cannot do absolute imports with ts-node either. We can install `tscong-paths` and add  `-r tsconfig-paths/register` here.